sdk/identity/identity/src/credentials/authorizationCodeCredential.ts

```diff
@@ -189,6 +192,7 @@ export class AuthorizationCodeCredential implements TokenCredential {
         code,
         message: err.message
       });
+      logger.getToken.error(err);
```

Comment:
I believe in other languages we're logging these errors at the info level still, since often they will not be critical errors. Critical errors will all be raised in exceptions thrown from the client generally. @catalinaperalta this is the case correct?

Reply:
@catalinaperalta let me know what's your approach and I'll adhere.

Reply:
Yes, I agree with Scott. We decided to only raise critical errors to the error level and leave others at the informational level.